prime-router/metadata/fhir_mapping/hl7/codesystem/ExtensionUrlMapping.yml

```diff
@@ -370,25 +377,22 @@
 # Comment type
 - id: "note-type"
   url: "https://reportstream.cdc.gov/fhir/StructureDefinition/note-type"
-# Identifier type
-- id: "identifier-type"
-  url: "https://reportstream.cdc.gov/fhir/StructureDefinition/identifier-type"
 
 ## Multipurpose
 # internal indicator of sender, usable for filtering
 - id: "sender-id"
   url: "https://reportstream.cdc.gov/fhir/StructureDefinition/sender-id"
-# TODO: collapse these into one
-# HL7 v2 Name
-- id: "hl7v2Name"
-  url: "https://reportstream.cdc.gov/fhir/StructureDefinition/hl7v2Name"
-# HL7 Use
+- id: "hl7v2Segment"
+  url: "https://reportstream.cdc.gov/fhir/StructureDefinition/hl7v2Segment"
+- id: "hl7v2Field"
+  url: "https://reportstream.cdc.gov/fhir/StructureDefinition/hl7v2Field"
+- id: "hl7v2Component"
+  url: "https://reportstream.cdc.gov/fhir/StructureDefinition/hl7v2Component"
+
+# Deprecated
 - id: "hl7-use"
```

Comment:
There is some usage of this in the old mappings and likely in the ORM mappings.  I'll open another ticket but I think it would make more sense to delete the deprecated mappings before trying to remove this, but I'm open to including it in this work.

Reply:
That sounds good, please link ticket in PR when created and optionally add comment here?